lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -1,15 +1,17 @@
 <%namespace name='static' file='../../static_content.html'/>
 
-<%! from django.utils.translation import ugettext as _
-import json
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 %>
+<%page expression_filter="h"/>
 
 <%static:require_module module_name="js/certificates/factories/certificate_whitelist_factory" class_name="CertificateWhitelistFactory">
-        CertificateWhitelistFactory('${json.dumps(certificate_white_list)}', "${generate_certificate_exceptions_url}", "${certificate_exception_view_url}", "${generate_bulk_certificate_exceptions_url}");
+        CertificateWhitelistFactory('${certificate_white_list | n, dump_js_escaped_json}', "${generate_certificate_exceptions_url | n, js_escaped_string}", "${certificate_exception_view_url | n, js_escaped_string}", "${generate_bulk_certificate_exceptions_url | n, js_escaped_string}");
```

Comment:
It's odd that the first argument is being dumped into a string, but you have done the right thing.  
Nit: you could update to single quotes throughout for consistency.